app/controllers/api/account.php

```diff
@@ -2369,15 +2370,13 @@
             throw new Exception(Exception::USER_SESSION_NOT_FOUND);
         }
 
-        $jwt = new JWT(System::getEnv('_APP_OPENSSL_KEY_V1'), 'HS256', 900, 10); // Instantiate with key, algo, maxAge and leeway.
```

Comment:
why are we removing the leeway?

Reply:
We now support 1-3600 seconds JWT keys. If developer decides to have JWT work for 15 seconds, it shouldn't be 25.